docs/static_site/src/pages/get_started/ubuntu_setup.md

```diff
@@ -63,11 +63,11 @@ Then download [cuDNN 7.1.4](https://developer.nvidia.com/cudnn).
 Unzip the file and change to the cuDNN root directory. Move the header and libraries to your local CUDA Toolkit folder:
 
 ```bash
-    tar xvzf cudnn-9.2-linux-x64-v7.1
```

Comment:
Why are we recommending cuda 9.2 in the install instructions?  Most are using 10.1 and 10.2

Reply:
This PR only changes formatting and spacing. For content change, please create a separate PR.